app/views/quotes/search.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Quotes Matching Search For: <%= "#{params[:query]}"%></h1>
```

Comment:
So we don't want to access the data that we have gotten back via hashes. That is why we would create a PORO or multiple POROs, so that we can create objects for our view to call/display.